src/pyrqlite/cursors.py

```diff
@@ -58,25 +58,26 @@ def close(self):
 
 
     def _request(self, method, uri, body=None, headers={}):
-        debug = logging.getLogger().getEffectiveLevel() < logging.DEBUG
-        logging.debug(
+        logger = logging.getLogger(__name__)
```

Comment:
The nice thing about doing it in local scope is that it doesn't pollute the modules global variables. The `logging.getLogger` function always returns the same instance for the same name, so the end result is roughly the same.